edn/tests/tests.rs

```diff
@@ -946,8 +944,8 @@ fn test_is_and_as_type_helper_functions() {
             value.is_map(),
         ];
 
-        for j in 0..values.len() {
-            assert_eq!(j == i, is_result[j]);
+        for (j, result) in is_result.iter().enumerate() {
```

Comment:
If you're going to do this, you should also `assert_eq!(values.len(), is_result.len())` outside this inner loop. Previously the array indexing bounds check would panic if these arrays were mis-sized.